new_app_setup.py

```diff
@@ -0,0 +1,266 @@
+                            cpu=cpu_requested,
+                            memory=ram_requested,
+                        )
+                    )
+                )
+            )
+        )
+# QUESTION: When running, I am getting the warning:
+
+# "error: error validating "test-team-name_pods.yml": error validating
+# data: ValidationError(Pod.spec.containers): invalid type for
+# io.k8s.api.core.v1.PodSpec.containers: got "map", expected "array";
+# if you choose to ignore these errors, turn validation off with
+# --validate=false
+# and I know that is because name does not have the hyphen indicating an array
+# in front of it, but I cannot seem to figure out how to do this...
+# anything I'm missing?
+
+    with open(pods_filename, 'w') as outfile:
+        yaml.dump(yml_file_kubernetes_data, outfile, default_flow_style=False)
+
+    print("The YAML file for {} pods were created and was "
+          "saved as {}".format(team_name, pods_filename))
+
+    return yml_file_kubernetes_data
+
+
+def create_kubernetes(filename, team_name, type_of_create):
+    subprocess.call(['kubectl', 'create', '-f', filename])
+
+    print("The Kubernetes {} for {} was "
+          "created.".format(type_of_create, team_name))
+
+
+def create_bucket(team_name):
+    pass
+    # export PROJECT_ID=$(gcloud config get-value team_name)
+    # storage_client = storage.Client()
+    # bucket = storage_client.create_bucket(team_name)
+    # print("Bucket {} was created.".format(bucket.name))
+
+
+def get_congif_yml_file(team_name):
+    config_filename = team_name+'_config_file.yml'
+
+    terminal_command = ['kubectl', 'config', 'view']
+
+    with open(config_filename, 'w') as outfile:
+        subprocess.call(terminal_command, stdout=outfile)
+
+    print("The config file for {} was created and "
+          "saved as {}".format(team_name, config_filename))
+    return config_filename
+
+
+def send_config_email_to_users(team_name, email_to, config_filename):
+    '''
+    Note: This code is going to change to reflect which email address we
+    would be sending this from. This successfully sends emails to GMAIL
+    and UOREGON accounts, I have not tried to send to any
+    other emails.
+    '''
+    # QUESTION: What email address do you want me to have these
+    # config files send from?
+    MVP_email_address = 'abkelley97@gmail.com'
+
+    # This line will be the actual one used to reflect the password
+    # for whatever email we send from.
+    # MVP_email_address_password = os.environ.get
+    #                               ('MVP_EMAIL_ADDRESS_PASSWORD')
+
+    # TEMP for my testing right now:
+    MVP_email_address_password = os.environ[str('PROF_EMAIL_PASS')]
+
+    email_contents_subject = ("Kubernetes config file for "
+                              "Team " + team_name + " at Hack For A Cause")
+
+    email_contents_body = ("Hi there,\nAttached to this email is your "
+                           "Kubernetes configuration file for your team " +
```

Comment:
the config file they will need is the one that contains their credentials which we probably don't want to send via email

Reply:
So should I delete all of the email information and not send anything to the team until later on when we focus on Step #7?

Reply:
I think that's right.